src/numberDisplayFormat.ts

```diff
@@ -169,7 +169,17 @@ function getDpFormat(
  * @param {string | BigNumber} value a value in string or in WAD
  * @returns {string} a millified value with 1 d.p. For example, 12.2M, 1.4K
  */
-function getMillifiedFormat(value: string | BigNumber): string {
+function getMillifiedFormat(
+  value: string | BigNumber,
+  shownLessThanZeroPointZeroOne = false
+): string {
+  const valueStr = getStringInput(value);
+  if (
+    shownLessThanZeroPointZeroOne &&
+    strToWad(valueStr).lt(strToWad("0.01"))
```

Comment:
in this case, `0` will be shown as `< 0.01` but we may want to show `0` instead.

Reply:
should add a test case for the situation where the value is 0